app/todo_lists/resources.py

```diff
@@ -0,0 +1,71 @@
+import json
+
+from flask import request
+from flask_restful import Resource
+
+from app import db, TodoList
+from app.lib.response_util import buildOkResponse
+
+
+def _returnTodoList(todoList):
+    """Private method to convert a todoList object into a dictionary.
+    """
+    return {
+        "id": todoList.id,
+        "name": todoList.name
+    }
+
+
+class TodoListMultiResource(Resource):
+    """Class for creating and accessing lists.
+    """
+    def get(self):
+        """Method gets a list of todoLists and returns them in an Ok response.
+        """
+        todoLists = TodoList.query.all()
+
+        return buildOkResponse([_returnTodoList(todoList) for todoList in
+                               todoLists])
+
+    def post(self):
+        """Method creates a todoList and returns it in an Ok response.
+        """
+        newListData = json.loads(request.form['data'])
+        newList = TodoList(newListData['name'], newListData['userId'])
+
+        db.session.add(newList)
+        db.session.commit()
+
+        return buildOkResponse(_returnTodoList(newList))
+
+
+class TodoListResource(Resource):
+    """Class to get, update, or delete a single todoList.
+    """
+    def put(self, listId):
+        """Method that updates and returns a todoList in an Ok response.
```

Comment:
listId is pretty self explanatory, but lets be consistent and add an Args section